### DISCUSSION TITLE
How to add items to a path where the key contains hyphens?

### DISCUSSION THREAD

**Discussion by rapkin61** [luqasz/librouteros#154] [Q&A]
>>> **import librouteros**
>>> **api = connect( username='admin', password='************', host='xxx.xxx.xxx.xxx',)**
>>> **capsman_interfaces = api.path('caps-man', 'interface')**

>>> **capsman_interfaces.add( name='test', radio-mac="08:00:00:00:00:08" )**
  File "<stdin>", line 1
    capsman_interfaces.add( name='test', radio-mac="08:00:00:00:00:08" )
                                         ^^^^^^^^^^
SyntaxError: expression cannot contain assignment, perhaps you meant "=="?

>>> **capsman_interfaces.add( name='test', "radio-mac"="08:00:00:00:00:08" )**
  File "<stdin>", line 1
    capsman_interfaces.add( name='test', "radio-mac"="08:00:00:00:00:08" )
                                         ^^^^^^^^^^^^
SyntaxError: expression cannot contain assignment, perhaps you meant "=="?

just to see, if add() works at all:

>>> **capsman_interfaces.add( name='test' )**
'*B'

that looks good!

but as soon as I try this with the radio-mac, it fails:

>>> **capsman_interfaces.add(  radio-mac='08:00:00:00:00:08' )**
  File "<stdin>", line 1
    capsman_interfaces.add(  radio-mac='08:00:00:00:00:08' )
                             ^^^^^^^^^^
SyntaxError: expression cannot contain assignment, perhaps you meant "=="?

What is my mistake here ?

**Comment by luqasz** [comment]
https://librouteros.readthedocs.io/en/3.2.1/path.html#update
